doc/release-notes/release-notes-4.0.0.md

```diff
@@ -0,0 +1,39 @@
+zend v4.0.0-RC2
+=========
+
+This is a release candidate of `zend` meant to be used primarily on testnet.
+As the previous version of `zend` (3.2.1), this RC1 is expected to deprecate at block **#1328320**, which will occur on **April 7th, 2023 at approximately 12:00 PM UTC**. Please, update to `v4.0.0` before April 7th (the final release will be available soon).
+
+A **hard fork** is expected to activate on testnet at block **#1228700**, which will occur on **March 13th at approximately 4:00 PM**.
```

Comment:
- We could remove the disclaimer to only use on testnet as it should be safe to use on mainnet
- Update with 3.3.1 and new deprecation height/date.
- Testnet hard fork is now in the past

Reply:
Force pushed after modifying as per your suggestion.